client/my-sites/plans/jetpack-plans/product-lightbox/product-select.tsx

```diff
@@ -51,7 +51,7 @@ const ProductSelect: FC< ProductSelectProps > = ( { product, siteId, onChangePro
 			<div className="product-lightbox__variants-options">
 				<MultipleChoiceQuestion
 					name="product-variants"
-					question={ PRODUCT_OPTIONS_HEADER[ product?.productSlug ] }
+					question={ PRODUCT_OPTIONS_HEADER[ product?.productSlug ] as string }
```

Comment:
I see we use `.toString()` in the same file in one of the above lines. Let's maybe unify the approach?

Reply:
Thanks, I didn't notice that because the `variantOptions` were copied over from the lightbox file. Nice catch!